packages/compartment-mapper/src/node-modules.js

```diff
@@ -626,13 +626,14 @@ const translateGraph = (
         const localPath = join(dependencyName, exportPath);
         if (
           !policy ||
-          dependencyAllowedByPolicy(
-            {
-              name,
-              path,
-            },
-            /** @type {import('./types.js').PackagePolicy} */ (packagePolicy),
-          )
+          (packagePolicy &&
```

Comment:
if `policy` is truthy and `packagePolicy` is falsy, we're under attack. should we introduce an assert before the if to make sure we explode immediately?

Reply:
@naugtur I've added an assertion to this effect